django/otto/celery.py

```diff
@@ -32,6 +32,21 @@
         "task": "otto.tasks.reset_weekly_bonus",
         "schedule": crontab(hour=0, minute=0, day_of_week=0),
     },
+    # Delete old chats (90 days retention) every day at 2 am UTC
+    "delete-old-chats-every-morning": {
+        "task": "otto.tasks.delete_old_chats",
+        "schedule": crontab(hour=2, minute=0),
+    },
+    # Delete empty chats every day at 2 am UTC
+    "delete-empty-chats-every-morning": {
+        "task": "otto.tasks.delete_empty_chats",
+        "schedule": crontab(hour=2, minute=0),
+    },
+    # Delete unused libraries every day at 3 am UTC
+    # "delete-unused-libraries-every-morning": {
```

Comment:
I am not sure this is a good idea, but I've added it as a comment so that it is clear we have decided not to do this.